source/content/guides/secrets/06-troubleshooting.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Pantheon Secrets Guide
+subtitle: Troubleshooting
+description: Securely store secrets in the Pantheon Platform.
+terminuspage: true
+contributors: [stovak]
+contenttype: [guide]
+innav: [true]
+categories: [secrets]
+cms: [drupal, wordpress]
+audience: [development]
+product: [secrets]
+integration: [--]
+tags: [reference, cli, local, terminus, workflow]
+permalink: docs/guides/secrets/troubleshooting
+reviewed: "2024-05-01"
+---
+## Integrated Composer Build fails on private packages
+
+This is the most common error when sites are using secrets and Integrated Composer. This may manifest in different ways and may be caused by different problems. This playbook tries to cover them.
+
+**Error getting a private package during the IC build**
+
+This is the most common error; an example message associated to this error is:
+
+- Failed to download vendor/package from dist (where vendor/package may be any private package)
+
+Some possible causes for this error:
+
+<dl>
+<dt>
+
+**Problem:** Secrets are not correctly set for the site. Secrets for IC should be of scope ic and type composer. See instructions in the plugin README: [https://github.com/pantheon-systems/terminus-secrets-manager-plugin#mechanism-1-oauth-composer-authentication](https://github.com/pantheon-systems/terminus-secrets-manager-plugin#mechanism-1-oauth-composer-authentication)
+
+</dt>
+<dd>
+
+**Solution:** ask the client to delete and recreate the secret if scope and type do not match for the given secret name
+
+</dd>
+</dl>
+
+<dt>
+
+**Problem:** Secret value/token may be expired
+
+</dt>
+<dd>
+
+**Solution:** ask the client to set the secret again to an updated value
+
+</dd>
+
+<dt>
+
+Problem: Site may be running on a PHP version < 8. If this is the case, there will be a message in the job output: “Skipping setting up secrets as it is not supported in PHP < 8”
```

Comment:
Inconsistent use of bold/unbold for the Problems/Solutions. They should probably all be **bold**.